Connectors/Autocad/Speckle.Connectors.AutocadShared/Bindings/AutocadBasicConnectorBinding.cs

```diff
@@ -116,42 +116,46 @@ private void HighlightObjectsOnView(ObjectId[] objectIds, string? modelCardId =
   {
     var doc = Application.DocumentManager.MdiActiveDocument;
 
-    Parent.RunOnMainThread(() =>
-    {
-      try
-      {
-        doc.Editor.SetImpliedSelection(Array.Empty<ObjectId>()); // Deselects
-        doc.Editor.SetImpliedSelection(objectIds); // Selects
-        doc.Editor.UpdateScreen();
-
-        Extents3d selectedExtents = new();
-
-        var tr = doc.TransactionManager.StartTransaction();
-        foreach (ObjectId objectId in objectIds)
+    Parent.RunOnMainThread(
+      () =>
+        Parent.TopLevelExceptionHandler.CatchUnhandled(() =>
```

Comment:
This is unnecessary because it is wrapped on the function itself already

https://github.com/specklesystems/speckle-sharp-connectors/blob/dimitrie/cnx-14-send-dynamic-blocks-from-acad-as-exploded-groupsstatic/DUI3/Speckle.Connectors.DUI/Bridge/BrowserBridge.cs#L182

Reply:
nice, will change!